public-images/public-images-how-to/launch-qcow-with-qemu.rst

```diff
@@ -0,0 +1,129 @@
+.. _qcow-qemu:
+
+Launch QCOW Images Using QEMU
+=============================
+
+Ubuntu cloud images are released in many formats to enable many launch configurations and methods.
+
+Among the many formats for cloud images is the ``.img`` QCOW format. This article will cover the QCOW use case and provide
+instructions on how to use the images with QEMU.
+
+For starting with an ISO server image, see the `server documentation`_ on downloading and converting an ISO for use
+with QEMU.
+
+QEMU and QCOWs
+--------------
+
+`QEMU`_ is a system emulator and "type 2" virtual machine hypervisor. Type 2 hypervisors run on top of the host operating
+system. Often, QEMU is used with `KVM`_, which is a Type 1 hypervisor, which can run directly on bare metal. Together,
+QEMU and KVM can enable running virtual machines at near-native speeds.
+
+QCOW, which stands for "QEMU Copy On Write," is a format for images used by QEMU. QCOW image files grow according to use,
+as opposed to pre-allocating the entire image disk to a file. Ubuntu cloud images with the ``.img`` file type are QCOW
+images and are best used with the QEMU hypervisor.
+
+QEMU Shell
+~~~~~~~~~~
+
+When launching a system it is handy to know that ctrl-a c is an escape key that will take the user from the monitor or
+serial console and to the QEMU monitor. From this shell a user can send keys to the guest via the sendkey option or even
```

Comment:
```suggestion
serial console of the local machine to the QEMU monitor. From this QEMU shell a user can send keys to the guest via the sendkey option or even
```
Are these suggestions correct? If not, could you try to clarify a bit?

Reply:
I've attempted to clarify.